extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-sidebar {
+  position: fixed !important;
+  top: 0 !important;
```

Comment:
It's possible. Do you think it'll be clear enough? This will also conflict slightly with your `height: 100vh;`. I think the only way to work it around would be by using `height: calc(100vh - {top offset value})`